test/blocks/aside/aside.test.js

```diff
@@ -19,13 +19,14 @@ describe('aside', () => {
 
       if (type !== 'notification') {
         it('has a heading', () => {
-          const heading = aside.querySelector('[class^=heading-]');
-          expect(heading).to.exist;
+          if (type !== 'promobar') {
+            const heading = aside.querySelector('[class^=heading-]');
+            expect(heading).to.exist;
+          }
```

Comment:
Won't this technically test only for if the type is not a `notification` and not a `promobar`, but it won't cover if it is just not a `notification` and just not a `promobar`? In other words, don't we lose a test for notification?

Reply:
Fixed the unit test. This will be checking the heading in promobar as well hence this would not be excluded for promobars.